golang/googleOauth/main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"golang.org/x/oauth2"
+	"golang.org/x/oauth2/google"
+)
+
+var (
+	googleOauthConfig *oauth2.Config
+	oauthStateString  = "pseudo-random"
+)
+
+func init() {
+	googleOauthConfig = &oauth2.Config{
+		RedirectURL:  "http://localhost:8080/callback",
+		ClientID:     "YOUR_CLIENT_ID",
+		ClientSecret: "YOUR_CLIENT_SECRET",
+		Scopes:       []string{"https://www.googleapis.com/auth/userinfo.email"},
+		Endpoint:     google.Endpoint,
+	}
```

Comment:
We should use environment variables to load the client id and secret here , please change this

Reply:
@44t4nk1 also mention in the issue that you are willing to contribute to google  golang oauth

Reply:
Okay